src/dbwrapper.cpp

```diff
@@ -10,7 +10,7 @@
 #include <leveldb/cache.h>
 #include <leveldb/env.h>
 #include <leveldb/filter_policy.h>
-#include <memenv.h>
+#include <leveldb/helpers/memenv/memenv.h>
```

Comment:
This feels kinda ugly, somehow. Some kind of abstraction leak (where leveldb is relative to the bitcoin core source code shouldn't matter, from the point of the code). Are you really sure this is needed?

Reply:
wait, no, it's not as bad as I thought. This is relative to leveldb's include directory, not the current source dir. Never mind.

Reply:
> This is relative to leveldb's include directory, not the current source dir.

The latter, I guess.

Reply:
> Are you really sure this is needed?

No. It is not strictly needed. A workaround can be used in the global CMake project.